content/docs/1.0.x/install/authenticate-cli-bridge/index.md

```diff
@@ -18,9 +18,15 @@ After [installing Keptn](../helm-install), you already have your Keptn endpoint.
 
 
 Get the Keptn endpoint from the `api-gateway-nginx`.
-If you are using port-forward to expose Keptn,
+The specifics for getting the Keptn endpoint vary slightly
+depending on the access option you are using for Keptn.
+Instructions for getting the Keptn endpoint and, optionally,
+storing that endpoint in an environment variable
+are detailed for each option in [Choose access options](../access).
+
+For example, if you are using `LoadBalancer` to expose Keptn,
```

Comment:
This paragraph doesn't make sense to me. You're talking about a load balancer endpoint here. But then you're providing localhost and port as they would be in a port forwarding setup. And then after that you show the api-gateway-nginx service which is configured in `ClusterIP` mode and not `LoadBalancer` mode. That service can only have an external IP if it's set to `LoadBalancer` mode. Never in `ClusterIP` mode.